includes/class-custom-post-type-fund.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * The core plugin class file
+ *
+ * Defines the functions necessary to register our custom post types with
+ * WordPress.
+ *
+ * @link       https://premium.wpmudev.org/blog/create-wordpress-custom-post-types/
+ * @since      1.0.0
+ *
+ * @package    WSUWP_Plugin_iDonate_Post_Type_Fund
+ * @author     Blair Lierman
+ */
+class WSUWP_Plugin_iDonate_Post_Type_Fund {
+
+	/**
+	 * Initializes the plugin by registering the hooks necessary
+	 * for creating our custom post type within WordPress.
+	 *
+	 * @since    1.0.0
+	 */
+	public function init() {
+
+		add_action( 'init', array( $this, 'custom_post_types' ) );
+
+	}
+
+	function custom_post_types() {
+		$labels = array(
+			'name'				=> 'Funds',
```

Comment:
While code style asks for tabbed indents, spaces should be used in between things like array key assignment. Otherwise alignment can get a little crazy. It doesn't hurt anything, just nitpicky. :)